mirdata/datasets/tonality_classicalDB.py

```diff
@@ -0,0 +1,252 @@
+        self.title = self.audio_path.replace(".wav", "").split("/")[-1]
+
+    @utils.cached_property
+    def key(self):
+        """String: key annotation"""
+        return load_key(self.key_path)
+
+    @utils.cached_property
+    def spectrum(self):
+        """np.array: spectrum"""
+        return load_spectrum(self.spectrum_path)
+
+    @utils.cached_property
+    def HPCP(self):
+        """np.array: HPCP"""
+        return load_HPCP(self.HPCP_path)
+
+    @utils.cached_property
+    def mb_metadata(self):
+        """Dict: musicbrainz metadata"""
+        return load_mb(self.mb_path)
+
+    @property
+    def audio(self):
+        """(np.ndarray, float): audio signal, sample rate"""
+        return load_audio(self.audio_path)
+
+    def to_jams(self):
+        """Jams: the track's data in jams format"""
+        return jams_utils.jams_converter(
+            audio_path=self.audio_path,
+            metadata={
+                "title": self.title,
+                "key": self.key,
+                "spectrum": self.spectrum,
+                "HPCP": self.HPCP,
+                "musicbrainz_metatada": self.mb_metadata,
+            },
+        )
+
+
+def load_audio(audio_path):
+    """Load a Tonality classicalDB audio file.
+
+    Args:
+        audio_path (str): path to audio file
+
+    Returns:
+        y (np.ndarray): the mono audio signal
+        sr (float): The sample rate of the audio file
+
+    """
+    if not os.path.exists(audio_path):
+        raise IOError("audio_path {} does not exist".format(audio_path))
+    return librosa.load(audio_path, sr=None, mono=True)
+
+
+def load_key(keys_path):
+    """Load Tonality classicalDB format key data from a file
+
+    Args:
+        keys_path (str): path to key annotation file
+
+    Returns:
+        (str): loaded key data
+
+    """
+    if keys_path is None:
+        return None
+
+    if not os.path.exists(keys_path):
+        raise IOError("keys_path {} does not exist".format(keys_path))
+
+    with open(keys_path) as f:
+        key = f.readline()
+
+    return key.replace('\t', ' ').replace('\n', '')
+
+
+def load_spectrum(spectrum_path):
+    """Load Tonality classicalDB spectrum data from a file
+
+    Args:
+        spectrum_path (str): path to spectrum  file
+
+    Returns:
+        (np.array): loaded spectrum data
+
+    """
+    if spectrum_path is None:
+        return None
+
+    if not os.path.exists(spectrum_path):
+        raise IOError("spectrum_path {} does not exist".format(spectrum_path))
+
+    with open(spectrum_path) as f:
+        data = json.load(f)
+
+    spectrum = [list(map(complex, x)) for x in data['spectrum']]
+
+    return np.array(spectrum)
+
+
+def load_HPCP(HPCP_path):
+    """Load Tonality classicalDB HPCP feature from a file
+    Args:
+        HPCP_path (str): path to HPCP file
+
+    Returns:
+        (np.array): loaded HPCP data
+
+    """
+    if HPCP_path is None:
+        return None
```

Comment:
Codecov is complaining about this line as it's not covered in the tests

Reply:
I had problems copy pasting